src/services/change-requests.ts

```diff
@@ -32,6 +35,10 @@ export const changeRequestTransformer = (changeRequest: ChangeRequest) => {
   return output;
 };
 
+export const allChangeRequestTransformer = (changeRequests: ChangeRequest[]) => {
```

Comment:
Why does this function need to be exported? I might have missed where it is used, but I'm pretty sure that keeping it inline like it was before is better.